webapp/src/pages/eventConfig/components/EventConfigComponent.js

```diff
@@ -271,12 +289,102 @@ class EventConfigComponent extends Component {
         isValid: errors.length === 0
       });
     });
+    console.log(this.state.updatedEvent);
   }
 
   getFieldNameWithLanguage = (input, lang) => {
     return input + " in " + lang;
   }
 
+  setRequiredDateFields = (event_type) => {
+    //could shift this const globally?
+    const requiredDateFields_by_event = {
```

Comment:
not sure if this const should be made global?

Reply:
Yeah would make sense to move it up to the top of the file :)